### ISSUE TITLE
Missing `form-data` dependency in `package.json`

### ISSUE THREAD

**Issue by raveclassic** [sendbird/SendBird-SDK-JavaScript#142]
`SendBird.min.js` directly requires package `form-data` in its dependencies but this package is not declared in `package.json` dependencies.
Quick search in git history shows that it was declared there once but then was removed.

**Comment by thierryskoda** [comment]
@raveclassic Did you also had the problem `Can't resolve 'form-data'` in `./node_modules/sendbird/SendBird.min.js` ?

**Comment by raveclassic** [comment]
@thierryskoda Exactly. We have this module installed as a transitive dependency in dev mode but with `NODE_ENV=production` it's not installed and the build fails.

**Comment by mirkogeest** [comment]
Same here, we had to add the package form-data to package.json to make it work at production
